pages.de/linux/snap.md

```diff
@@ -0,0 +1,12 @@
+# snap
+
+> SNAP (Sentinel Application Platform) für die Prozessierung von Satellitendaten der Europäischen Raumfahrtagentur (ESA).
+> Download: <http://step.esa.int/main/>.
+
+- Anzeigen aller Parameter:
```

Comment:
We write descriptions in imperative mood. Also for the `--help flag`, we usually use something like this:
```suggestion
- Zeige Hilfe an:
```